### ISSUE TITLE
Error in file causes unhandled promise rejection

### ISSUE THREAD

**Issue by Jameskmonger** [alsatian-test/alsatian#421]
If a file throws an error when being required, such as a `ReferenceError`, there is no output from alsatian - it fails silently.

In `test-loader`, we need to check for an error being thrown from the `require` call, something like this:

```
var testFixureModule;

try {
    testFixureModule = this._fileRequirer.require(filePath);
} catch (e) {
    // error here shows that the file had a load error
}
```

**Comment by Jameskmonger** [comment]
Temporary fix for this in #422. It isn't silent anymore but we need to investigate and put a proper fix in.

**Comment by jhm-ciberman** [comment]
+1 Can confirm this issue in my own project

**Comment by Jameskmonger** [comment]
Solved in #422 (not a temporary fix)

**Comment by jamesrichford** [comment]
@jhm-ciberman the fix (to give proper information about the failure) should be available in 2.1.0 now
